sdcm/sct_config.py

```diff
@@ -1693,6 +1694,8 @@ def _validate_nemesis_can_run_on_non_seed(self) -> None:
 
     def _check_per_backend_required_values(self, backend: str):
         if backend in self.available_backends:
+            if backend in ('aws', 'gce') and self.get("db_type") == "cloud_scylla":
+                backend += "-siren"
```

Comment:
This logic contradicts the idea of removing "*siren" backends logic as part of SCT code clean up.

Reply:
List of mandatory variables for non-siren and siren backends are different and saves in the dictionary. If remove "siren" we need to change this approach

Reply:
I know that it is different I just wanted to say that we should not stick to the backend names we which we are going to remove.